frontend/src/components/Project.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="col project table-responsive">
+  <div class="project table-responsive">
```

Comment:
So did this clear up the "squishing" issue?

Reply:
This part fixed each project's table being on the same horizontal row. Looks good on chrome at least.. I guess I should check others.